srcpkgs/python3-saml2/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'python3-saml2'
 pkgname=python3-saml2
 version=7.4.2
-revision=2
+revision=3
```

Comment:
what? there is not shlib dependency on anyhing

Reply:
I agree with you. However, take a look at the build failure now that I've dropped this commit from the branch. It never builds xmlschema prior to building synapse even though xmlschema is a transitive check dependency.

Reply:
Here's [the test failure log](https://gist.github.com/TinfoilSubmarine/a35c0045f0e20f60e073d5cebe8e7ee4), I added the commit back so I can continue testing.

Reply:
i pushed c9325d633ebbacf473c8c90a7caca3ebf15d5a2e so it shouldn't be an issue